packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -323,7 +244,7 @@ The link points to your Okta org as in: `https://yourorg.okta.com/email/verify/0
 
 When the user clicks the magic link, your org receives the request, gets the OTP and state parameters, and forwards the request with these parameters to your application. The org combines the Callback URI that you defined in [Update configurations](#update-configurations) with the OTP and state parameters to produce a final callback URL for the user. For example, `https://localhost:44314/magiclink/callback?otp=726009&state=1b34371af02dd31d2bc4c48a3607cd32`
 
-### 10. Handle the magic link redirect in your app
+### 8. Handle the magic link redirect in your app
```

Comment:
The diagram lists this as "8. Handle redirect in your app". Make this text consistent with the diagram.